### PR TITLE
Fix #303

### LINKED ISSUE #303: omf install removes symlinked bundle
I use a dotfile manager that symlinks `$OMF_CONFIG/bundle`. When I run `omf install <plugin-name>` the link gets replaced by a file. Many dotfile managers use symlinks. It would be useful to change this behavior.

### PR DESCRIPTION
**Pull request by andrewrynhard**
This PR adds the ability to set `bundle` appropriately in the case that it is a symlink.

### CODE REVIEW

File: pkg/omf/functions/bundle/omf.bundle.add.fish [hunk 1]
```diff
@@ -1,5 +1,10 @@
 function omf.bundle.add -a type name_or_url
-  set -l bundle $OMF_CONFIG/bundle
+  if test -h $OMF_CONFIG/bundle
```
Comment (oranja) [inline pkg/omf/functions/bundle/omf.bundle.add.fish]: I'm looking both at the fish website and the man page, and `-h` is undocumented. Maybe `-L` would make it easier for people to understand what it's doing?

File: pkg/omf/functions/bundle/omf.bundle.add.fish [hunk 2]
```diff
@@ -1,5 +1,10 @@
 function omf.bundle.add -a type name_or_url
-  set -l bundle $OMF_CONFIG/bundle
```
Comment (oranja) [inline pkg/omf/functions/bundle/omf.bundle.add.fish]: By removing the `-l` flag, you allow `$bundle` to clutter the user's environment.
I suggest keeping this line as default, and then you can change it inside the first `if` case, only if it's a symlink. The second case is then irrelevant.

**Comment by oranja** [comment]
I added some comments on the code, but overall this is even better than what I had in mind.
Thank you!

File: pkg/omf/functions/bundle/omf.bundle.add.fish (hunk 2 above)
Comment (andrewrynhard) [inline pkg/omf/functions/bundle/omf.bundle.add.fish]: `-l` was not used in `omf.bundle.remove.fish`. Would you like the `-l` flag there as well?

Comment (oranja) [inline pkg/omf/functions/bundle/omf.bundle.add.fish]: Yes, please.
It's always good to have a new pair of eyes looking at the code. :)

**Comment by andrewrynhard** [comment]
Let me know if I can change anything else.

**Comment by oranja** [comment]
:+1: Looking good.
We'll wait to see what the others think. They are the ones with the commit permissions anyway... ;)

**Comment by derekstavis** [comment]
Cool! I like this fix, very clean and simple. Thanks for implementing!!

**Comment by coderstephen** [comment]
Elegant solution. :+1:

**Comment by derekstavis** [comment]
I think we can merge it :+1:

**Comment by bobthecow** [comment]
:+1:

**Pull request merged**